services/web/server/requirements/_test.txt

```diff
@@ -2,107 +2,107 @@
+openapi-core==0.12.0      # via -r requirements/_base.txt
+openapi-spec-validator==0.2.8  # via -r requirements/_base.txt, -r requirements/_test.in, openapi-core
 packaging==20.3           # via pytest, pytest-sugar
-pamqp==2.3.0              # via -r _base.txt, aiormq
-passlib==1.7.2            # via -r _base.txt
+pamqp==2.3.0              # via -r requirements/_base.txt, aiormq
+passlib==1.7.2            # via -r requirements/_base.txt
 pluggy==0.13.1            # via pytest
-prometheus-client==0.7.1  # via -r _base.txt
-psycopg2-binary==2.8.5    # via -r _base.txt, aiopg, sqlalchemy
-ptvsd==4.3.2              # via -r _test.in
+prometheus-client==0.7.1  # via -r requirements/_base.txt
+psycopg2-binary==2.8.5    # via -r requirements/_base.txt, aiopg, sqlalchemy
+ptvsd==4.3.2              # via -r requirements/_test.in
 py==1.8.1                 # via pytest
-pycparser==2.20           # via -r _base.txt, cffi
-pylint==2.5.0             # via -r _test.in
+pycparser==2.20           # via -r requirements/_base.txt, cffi
+pylint==2.5.0             # via -r requirements/_test.in
 pyparsing==2.4.7          # via packaging
-pyrsistent==0.16.0        # via -r _base.txt, jsonschema
-pytest-aiohttp==0.3.0     # via -r _test.in
-pytest-cov==2.8.1         # via -r _test.in
-pytest-docker==0.7.2      # via -r _test.in
-pytest-instafail==0.4.1.post0  # via -r _test.in
-pytest-mock==3.1.0        # via -r _test.in
-pytest-runner==5.2        # via -r _test.in
-pytest-sugar==0.9.3       # via -r _test.in
-pytest==5.3.5             # via -r _test.in, pytest-aiohttp, pytest-cov, pytest-instafail, pytest-mock, pytest-sugar
+pyrsistent==0.16.0        # via -r requirements/_base.txt, jsonschema
+pytest-aiohttp==0.3.0     # via -r requirements/_test.in
+pytest-cov==2.8.1         # via -r requirements/_test.in
+pytest-docker==0.7.2      # via -r requirements/_test.in
+pytest-instafail==0.4.1.post0  # via -r requirements/_test.in
+pytest-mock==3.1.0        # via -r requirements/_test.in
+pytest-runner==5.2        # via -r requirements/_test.in
+pytest-sugar==0.9.3       # via -r requirements/_test.in
+pytest==5.3.5             # via -r requirements/_test.in, pytest-aiohttp, pytest-cov, pytest-instafail, pytest-mock, pytest-sugar
 python-dateutil==2.8.1    # via faker
-python-engineio==3.12.1   # via -r _base.txt, python-socketio
-python-socketio==4.5.1    # via -r _base.txt
-pytz==2019.3              # via -r _base.txt, celery
-pyyaml==5.3.1             # via -r _base.txt, aiohttp-swagger, openapi-spec-validator
-redis==3.4.1              # via -r _test.in
+python-engineio==3.12.1   # via -r requirements/_base.txt, python-socketio
+python-socketio==4.5.1    # via -r requirements/_base.txt
+pytz==2020.1              # via -r requirements/_base.txt, celery
+pyyaml==5.3.1             # via -r requirements/_base.txt, aiohttp-swagger, openapi-spec-validator
+redis==3.5.0              # via -r requirements/_test.in
```

Comment:
This deprecates [hmset in the resrouce_manager/registry](https://github.com/pcrespov/osparc-simcore/blob/fbba9205f450f571bfceeada4684a2b0d66340e9/services/web/server/src/simcore_service_webserver/resource_manager/registry.py#L61) might want to keep an eye for it. [redis changelog](https://github.com/andymccurdy/redis-py/blob/252c840ea2ade01637b4ed7c834dd83b130041b4/CHANGES#L17-L18)

Reply:
Upon further investigation **aioredis** is used in the project. This is not an issue.
Redis is only used for testing (as the comment suggests and does not influence our code).